index.html

```diff
@@ -380,10 +380,10 @@
 		<div class="col-sm-2">Stanley Stevens&nbsp;&nbsp;<i class="fa fa-rocket"></i></div>
 		<div class="col-sm-2">
 			<a href="https://github.com/Stanleyyork" target="_blank"><i class="fa fa-github-square fa-2"></i></a> &nbsp;&nbsp;
-			<i class="fa fa-linkedin fa-2"></i> &nbsp;&nbsp;
-			<i class="fa fa-twitter fa-2"></i>&nbsp;&nbsp;
-			<i class="fa fa-angellist"></i>&nbsp;&nbsp;
-			<i class="fa fa-medium"></i>
+			<a href="https://linkedin.com/in/stanleyyork" target="_blank"><i class="fa fa-linkedin fa-2"></a></i> &nbsp;&nbsp;
```

Comment:
The closing </a> tag should be outside of the closing </i> tag so that the whole icon is a hyperlink. Please fix and update pull request.